mesh_tensorflow/transformer/transformer.py

```diff
@@ -547,6 +547,31 @@ def sublayer_rms_norm_subsampled(x, layer_stack, context, percentage=100.,
         mtf.square(var_activations), reduced_dim=var_dim)
   return x * mtf.rsqrt(variance + epsilon) * scale
 
+@gin.configurable
+def sublayer_scale_norm(x, layer_stack, context, epsilon=1e-6, name="scale_norm"):
```

Comment:
Seems like this is the same as sublayer_rms_norm. Am I missing something?

Reply:
@sharannarang they are basically the same, except that scale norm has just 1 parameer, while rms norm has parameter per each feature in feature dimension

Reply:
Ahh, thanks for pointing that out.